docs/vsphere_ci/scripts/install-firewall-rules.ps1

```diff
@@ -0,0 +1,16 @@
+# Powershell script to configure Firewall Rules in Windows Server 1809 and later.
+#
+# USAGE
+#    ./install-firewall-rules.ps1 
+
+# create firewall rule to allow Container Logs on port 10250
+New-NetFirewallRule -DisplayName "ContainerLogsPort" -LocalPort 10250 -Enabled True -Direction Inbound -Protocol TCP -Action Allow -EdgeTraversalPolicy Allow
+
+# check for OpenSSH-Server-In-TCP rule, it should be created automatically during setup
+if (-not($(Get-NetFirewallRule -DisplayName "OpenSSH Server (sshd)" | Get-NetFirewallPortFilter | Where-Object {($_.Protocol -eq "TCP") -and ($_.LocalPort -eq 22)}))) {
+    # If the firewall rule does not exist, create one
+    New-NetFirewallRule -DisplayName 'OpenSSH Server (sshd)' -LocalPort 22 -Enabled True -Direction Inbound -Protocol TCP -Action Allow 
+}
```

Comment:
Why is this being done given we were not doing this before?

Reply:
Glad you ask, since I have mixed feeling too. 

This rule should be created automatically by the OpenSSH server installation, it did fails sometime running local tests, so I add it explicitly.

Happy to remove it, and just rely on the OpenSSH installation process.

Reply:
I would then add to the ssh script with a comment mentioning that it sometimes does not created by the setup.

Reply:
makes sense, thank you